internal/migrate/versions.go

```diff
@@ -84,13 +83,13 @@ func (migration *Migration) Run(log *zap.Logger, db DB) error {
 	}
 
 	if version < 0 {
-		log.Info("Latest Version", zap.Int("version", version))
+		log.Info("Migration: Latest Version", zap.Int("version", version))
```

Comment:
remove the prefixes, they are not needed because the logger should already have the necessary prefix.